src/helm/benchmark/presentation/run_entries_speech.conf

```diff
@@ -1,16 +1,60 @@
 entries: [
```

Comment:
@ImKeTT  can you double check we're not missing anything?

Reply:
I think there are 9 tasks for now, I added `common_voice_15 ` and the `fleurs` tasks below. The `run_entries_speech.conf` should look like this:

```bash
entries: [
    ####################################################################################################################
    # Auditory Perception
    ####################################################################################################################
    {description: "meld_audio:model=audiolm", priority: 1}
    {description: "vocal_sound:model=audiolm", priority: 1}
    {description: "audiocaps:model=audiolm", priority: 1}

    ####################################################################################################################
    # Fairness
    ####################################################################################################################

    ####################################################################################################################
    # Robustness
    ####################################################################################################################

    {description: "audio_mnist:model=audiolm", priority: 1}

    {description: "speech_robust_bench:subject=accented_cv,model=audiolm", priority: 1}
    {description: "speech_robust_bench:subject=accented_cv_es,model=audiolm", priority: 1}
    {description: "speech_robust_bench:subject=chime_far,model=audiolm", priority: 1}
    {description: "speech_robust_bench:subject=chime_near,model=audiolm", priority: 1}
    {description: "speech_robust_bench:subject=ami_far,model=audiolm", priority: 1}
    {description: "speech_robust_bench:subject=ami_near,model=audiolm", priority: 1}


    ####################################################################################################################
    # Bias
    ####################################################################################################################


    ####################################################################################################################
    # Multilinguality
    ####################################################################################################################

    # TODO: populate with the rest of the languages
    {description: "covost2:source_language=English,target_language=Chinese,model=audiolm", priority: 1}

    # I think we can just use three langauges for multilingual_librispeech for now
    # There are other languages in the multilinguality aspect
    {description: "multilingual_librispeech:language=Dutch,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=Spanish,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=Italian,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=German,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=French,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=Portuguese,model=audiolm", priority: 1}
    {description: "multilingual_librispeech:language=Polish,model=audiolm", priority: 1}

    # TODO: populate with the rest of the languages
    {description: "fleurs:language=Finnish,model=audiolm", priority: 1}
    {description: "fleurs:language=Bulgarian,model=audiolm", priority: 1}
    {description: "fleurs:language=Hebrew,model=audiolm", priority: 1}
    {description: "fleurs:language=Zulu,model=audiolm", priority: 1}
    {description: "fleurs:language=Bengali,model=audiolm", priority: 1}
    {description: "fleurs:language=Thai,model=audiolm", priority: 1}
    {description: "fleurs:language=Mandarin_Chinese,model=audiolm", priority: 1}

    {description: "common_voice_15:language=English,model=audiolm", priority: 1}
    {description: "common_voice_15:language=Chinese_hk,model=audiolm", priority: 1}
    {description: "common_voice_15:language=German,model=audiolm", priority: 1}
    {description: "common_voice_15:language=French,model=audiolm", priority: 1}

    ####################################################################################################################
    # Toxicity
    ####################################################################################################################


    ####################################################################################################################
    # Safety
    ####################################################################################################################


]
```

Reply:
Also, I'm thinking of just using a subset of `multilingual_librispeech`, since some languages overlap with `common_voice_15 `. Maybe we can skip German and French for now?